tests/integration/dismiss-mem.tcl

```diff
@@ -11,6 +11,8 @@ start_server {tags {"dismiss external:skip"}} {
     # to satisfy the conditions for release memory pages, especially, we assume
     # the page size of OS is 4KB in some cases.
     test {dismiss all data types memory} {
+        r config set appendonly yes
+        waitForBgrewriteaof r
```

Comment:
in this test there's only one server serving one test, we can let it have `appendonly` enabled on startup, and there's no need for `waitForBgrewriteaof`. also no need to disable aof at the end.